src/building-blocks/FormItem.tsx

```diff
@@ -97,13 +98,15 @@ class FormItem extends Component<IFormFieldProps> {
   }
 
   public render() {
-    const { formManager, fieldConfig, layout } = this.props,
+    const { formManager, fieldConfig, layout, colon } = this.props,
       { colProps, formItemProps, field } = fieldConfig,
+      hasColon = !(colon === false || layout === 'vertical'),
       className = cx(
         FORM_ITEM_CLASS_NAME,
         fieldConfig.className,
         formItemProps && formItemProps.className,
         `${FORM_ITEM_CLASS_NAME}-${layout}`,
+        `${FORM_ITEM_CLASS_NAME}${hasColon ? '' : '-no'}-colon`,
```

Comment:
I think this bit of code is in 3 places now? We should probably group it into a util, something like:

```ts
function formatClassNames (className: string, layout, colon: boolean) {
  const hasColon = !(colon === false || layout === LAYOUT_TYPES.VERTICAL);
  return {
    `${FORM_ITEM_CLASS_NAME}-${layout}`,
    `${FORM_ITEM_CLASS_NAME}${hasColon ? '' : '-no'}-colon`,
  }
}
```
then `{...formatClassNames(FORM_ITEM_CLASS_NAME, layout, colon)}`

Reply:
Also we should have a const for the layout strings.